modules/ROOT/pages/microsoft-azure-sb-connector.adoc

```diff
@@ -0,0 +1,334 @@
+. Drag a *Record Variable*, *Session Variable*, or *Variable* component to the flow.
+. Store the *Lock Token* from the received message by using this expression: `#[message.inboundProperties.get('lockToken')]`
+. Do something with the message. For testing purposes, a simple *Logger* would suffice.
+. Add another *Microsoft Azure Service Bus* connector to the flow and set it to call the *Acknowledge message* operation. For the *Lock Token* field value, retrieve the stored *Lock Token*.
+
+*Reviewers: I see a Lock field but not a Lock Token field. So should I replace Lock Token with Lock. Also, when you say "Do something with the message", can you be more specific?*
+
+The connector processes every message received and acknowledges each message that is processed successfully.
+
+To have the connector automatically acknowledge messages, set the *Receive Mode* field to `AUTOMATIC`.
+
+[[scheduled]]
+=== Scheduled Reception of Messages
+
+The following example configures a Mule app that receives messages from a queue or topic on a specified schedule:
+
+Create a Mule app to receive messages from a queue or topic, as described in <<basic>>.
+
+When you create the app:
+
+* Set the *Unix CRON Expression* field to a CRON expression that follows the UNIX standard. For example, to connect and receive messages every day at 8 AM, use `0 8 * * *`.
+* Set the *Max. Messages to Receive* field to the maximum number of messages for the connector to receive every time the CRON expression triggers a receive.
+
+The connector executes a batch receive every time the CRON expression commands it. Any number of available messages up to the maximum set will be retrieved and processed by the flow.
+
+[[batch]]
+=== Queue Send Batch
+
+The following example configures a Mule app to send multiple messages to a queue:
+
+. Create a Mule app to receive messages from a queue or topic, as described in <<basic>>.
+. Replace the call to the *Publish message* operation with a call to the *Publish batch of messages* Operation. Make sure the *Destination type* and *Destination name* are properly set.
+. When crafting the message, be sure to create a list of strings, since the app will send many messages simultaneously.
++
+*Reviewers: what is the format of the list of strings? Is it comma-separated?*
++
+To split a string by the commas, and send every piece as an individual message, drag a *Transform message* and use the following script:
+```
+%dw 1.0
+%output application/java
+---
+payload splitBy ','
+```
+
+[[enqueuing]]
+=== Schedule the Logical Enqueuing of a Message
+
+The following example configures a Mule app to send a message to all listeners at a specified time:
+
+. Create a Mule app to receive messages from a queue or topic, as described in <<basic>>.
+. Fill in the "Scheduled enqueue time UTC" of the *Publish message* call parameter with the date/time you want the message to be dispatched. For example: `2019-06-27T21:16:46.866Z`.
```

Comment:
Fill in > Complete

ditto GLOBAL re. using articles before UI label names: need a common noun object

date/time > either "data and time" or "date or time," depending on which is most common or likely.